modules/hosted-control-planes-overview.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-overview_{context}"]
+= Hosted control planes overview (technology preview)
+
+Hosted control planes for {product-title} provides a way to decouple your control plane from your worker nodes so that you can work in a hybrid cloud approach. You can enable hosted control planes through the multicluster engine for Kubernetes operator version 2.0. In that version, only Amazon Web Services (AWS) hosted clusters are available.
+
+:FeatureName: Hosted control planes
```

Comment:
It's actually "hosted control planes" all lower-case but when starting a sentence it gets tricky :)

Reply:
I can try to rewrite any sentences that begin with "hosted control planes".

Reply:
You could say "Support for hosted control planes"